<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="hp-H310B" failures="1" tests="21" name="testcases_package.sanityAfterLogin" time="105.123" errors="1" timestamp="2024-05-06T17:00:06 IST" skipped="0">
  <testcase classname="testcases_package.sanityAfterLogin" name="verify_try_now_button_is_functional" time="1.181"/>
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="verify_try_now_button_is_functional2" time="0.945"/>
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="verify_dropdown_values_Static_Residential_ProxiesISPplus" time="2.660"/>
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="verify_dropdown_values_Static_Residential_Proxies" time="3.220"/>
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="verify_dropdown_values_Dedicated_Datacenter_Proxies" time="2.684"/>
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="verify_dropdown_values_Shared_Datacenter" time="1.929"/>
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="verify_dropdown_navigate_to_my_proxies_page" time="0.895"/>
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="verify_dropdown_navigate_to_my_profile" time="1.026"/>
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="verify_dropdown_navigate_to_my_Payment_method" time="5.072"/>
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="Verify_that_the_ISP_product_has_been_successfully_purchased" time="20.647"/>
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="Verify_Proxie_configuration_page_should_be_oppen" time="21.468">
    <error message="Expected condition failed: waiting for visibility of element located by By.xpath: (//span[text()=&#039; Configure &#039;])[2] (tried for 20 second(s) with 500 milliseconds interval)" type="org.openqa.selenium.TimeoutException">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for visibility of element located by By.xpath: (//span[text()=' Configure '])[2] (tried for 20 second(s) with 500 milliseconds interval)
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:95)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:272)
at testcases_package.sanityAfterLogin.Verify_Proxie_configuration_page_should_be_oppen(sanityAfterLogin.java:359)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1207)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"(//span[text()=' Configure '])[2]"}
  (Session info: chrome=123.0.6312.105)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'hp-H310B', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '6.2.0-39-generic', java.version: '17.0.9'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 123.0.6312.105, chrome: {chromedriverVersion: 123.0.6312.122 (31f8248cdd9..., userDataDir: /tmp/.org.chromium.Chromium...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:39407}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: LINUX, platformName: LINUX, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 6dcd9606b218e0698b5af45285d231f6
*** Element info: {Using=xpath, value=(//span[text()=' Configure '])[2]}
at jdk.internal.reflect.GeneratedConstructorAccessor12.newInstance(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:428)
at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:205)
at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:201)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:249)
... 30 more
]]>
    </error>
  </testcase> <!-- Verify_Proxie_configuration_page_should_be_oppen -->
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="verify_Documentation_Link_Is_Functional" time="9.390"/>
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="verify_Country_Dropdown_Is_Functional_under_configure_proxy_page" time="3.413"/>
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="verify_Change_Payment_Method_Update_Is_Functional_under_configure_proxy_page" time="5.857">
    <failure message="expected [Your payment method has been updated for this order] but found [Your payment method has been updated for this order.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Your payment method has been updated for this order] but found [Your payment method has been updated for this order.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at testcases_package.sanityAfterLogin.verify_Change_Payment_Method_Update_Is_Functional_under_configure_proxy_page(sanityAfterLogin.java:592)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1207)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verify_Change_Payment_Method_Update_Is_Functional_under_configure_proxy_page -->
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="verify_navigate_to_Change_Payment_Method_Is_Functional_under_configure_proxy_page" time="4.290"/>
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="verify_History_Is_Functional_under_configure_proxy_page" time="4.136"/>
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="verify_Auto_renew_Setting_Page_Is_Functional_under_configure_proxy_page" time="2.029"/>
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="verify_Add_Page_Is_Functional_under_configure_proxy_page" time="1.553"/>
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="verify_Your_payment_method_has_been_updated_for_this_order" time="5.132"/>
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="verify_dropdown_navigate_to_my_Payment_method_update" time="4.839"/>
  <system-out/>
  <testcase classname="testcases_package.sanityAfterLogin" name="verify_update_my_Profile" time="2.757"/>
  <system-out/>
</testsuite> <!-- testcases_package.sanityAfterLogin -->
